### PR TITLE
Exporter les membres d'un salon dans un fichier csv

### PR DESCRIPTION
**Pull request by aulamber**
Cf. https://github.com/tchapgouv/tchap-web-v4/issues/593

### CODE REVIEW

**Comment by estellecomment** [comment]
Code written. Tests coming. 

Changes : 
 - moved the button to the Personnes tab. 
 - used AccessibleTooltipButton instead of homemade tooltip code.
 - used FileDownloader instead of homemade download code.
 - renamed things in code
 - shortened user-facing text
 - filename is translated, and contains the name of the room (sanitized)

Francais :
When button is hovered :
image

When button is clicked : 
image

----
Anglais
When button is hovered :
image

When button is clicked : 
image
---
Content of downloaded text file (example) : 
`@prenom.nom-beta.gouv.fr:agent.dinum.tchap.gouv.fr,@autreprenom.autrenom-machin.gouv.fr:agent.interieur.tchap.gouv.fr`

File: cypress/e2e/export-room-members/export-room-members.spec.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        RoomUtils.createPublicRoom(roomName).then((roomId) => {
+            RoomUtils.openPeopleMenu(roomName);
+            cy.get('[data-testid="tc_exportRoomMembersButton"]')
+                .trigger("mouseover")
+                .get(".tc_exportRoomMembersTooltip"); // tooltip should show
+            cy.leaveRoom(roomId);
+        });
+    });
+
+    it("downloads the file when button is clicked", () => {
```
Comment (odelcroi) [inline cypress/e2e/export-room-members/export-room-members.spec.ts:33]: nice

File: patches/export-room-members/matrix-react-sdk+3.71.1.patch [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                     </React.Fragment>
+                 }
+                 footer={footer}
+@@ -443,4 +465,24 @@ export default class MemberList extends React.Component<IProps, IState> {
+             roomId: this.props.roomId,
+         });
+     };
++
++    /* TCHAP */
++    private onExportButtonClick = (ev: ButtonEvent): void => {
```
Comment (odelcroi) [inline patches/export-room-members/matrix-react-sdk+3.71.1.patch]: along with the button, could it have been a separated component to aliviate the patch footprint?

**Review by odelcroi — APPROVED** [review]

**Comment by odelcroi** [comment]
wonderful

**Pull request merged**